src/main/kotlin/org/digma/intellij/plugin/ui/list/insights/ScalingListViewItemsPanel.kt

```diff
@@ -0,0 +1,46 @@
+package org.digma.intellij.plugin.ui.list.insights
+
+import com.intellij.openapi.project.Project
+import org.digma.intellij.plugin.model.rest.insights.*
```

Comment:
try to get rid of the .* with specific classes

Reply:
ok I will provide more specific imports instead of generic one